### PR TITLE
Status Banner Color Updates

### PR DESCRIPTION
**Pull request by theilman7**
There are a few things that will need to change, but wanted to get this in to PR for initial feedback. Let me know how you would like the colors formatted in relation to the project. In addition, let me know if the markup I have makes sense. Thinking I could possibly encapsulate the elements I selected by using `*`... but let me know your thoughts.

### CODE REVIEW

File: src/styles/partials/components/_alert.scss [hunk 1]
```diff
@@ -79,16 +82,30 @@
...
+        color: $black;
+      }
+
+      .dg_alert__icon {
+        color: $black;
+      }
+
+      p {
+        color: $black;
+      }
```
Comment (martypowell) [inline src/styles/partials/components/_alert.scss]: i dont think you need nay of these lines if you use *

File: src/styles/partials/components/_alert.scss [hunk 2]
```diff
@@ -104,8 +121,22 @@
...
       outline-color: $error-color;
 
+      .dg_alert__status {
+        color: $black;
+      }
+
+      .dg_alert__icon {
+        color: $black;
+      }
+      * {
```
Comment (martypowell) [inline src/styles/partials/components/_alert.scss:115]: think you can just use * here

File: src/styles/variables/_colors.scss [hunk 3]
```diff
@@ -38,8 +38,11 @@ $gray-dark: mix(black, $gray, $gray-light-ratio);
 $gray-darkest: mix(black, $gray, $gray-lightest-ratio);
 $black: black;
 $white: white;
-
+$alice-blue: #E8F4FD;
```
Comment (martypowell) [inline src/styles/variables/_colors.scss]: i'd go with $status-blue or something like that.

Comment (martypowell) [reply src/styles/variables/_colors.scss]: i actually have mixed feelins about this, but i think htat naming convetion lets us know more about what these colors are, and to not really use them outside of their purpose.

**Review by martypowell — CHANGES_REQUESTED** [review]
a couple minor suggestions.

File: src/styles/variables/_colors.scss (hunk 3 above)
Comment (theilman7) [reply src/styles/variables/_colors.scss]: Updated. Ready for another look @martypowell

Comment (martypowell) [reply src/styles/variables/_colors.scss]: youll want to hit hit the other status colors too.

File: src/styles/partials/components/_alert.scss (hunk 2 above)
Comment (theilman7) [reply src/styles/partials/components/_alert.scss:115]: Did a sweep and cleaned up properties a bit. Take a look and let me know if this looks better. This should address your comment above as well. @martypowell

File: src/styles/variables/_colors.scss (hunk 3 above)
Comment (theilman7) [reply src/styles/variables/_colors.scss]: Did a big cleanup. Pushing up. @martypowell

**Review by martypowell — APPROVED** [review]

**Pull request merged**